### ISSUE TITLE
Elevated circulating L-alloisoleucine concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6042]
**Preferred term label:** Elevated circulating L-alloisoleucine concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of L-alloisoleucine in the blood is above the normal range., PMID: 10508118

**Parent term (use hpo.jax.org/app)** Hyperisoleucinemia HP:0010913

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Added as HP:0033155
